export.php

```diff
@@ -72,6 +72,6 @@
         generate_report_feedback_csv($moduleinstance->id, $mode);
     }
 } else {
-    $content = get_string('results_denied', 'digitala');
-    redirect($CFG->wwwroot.'/mod/digitala/export.php?id='.$moduleinstance->id.'&mode='.$mode);
+    redirect('view.php?id='.$id);
```

Comment:
Without `$CFG->wwwroot` this will fail in any environments with additional folder in path, for example in staging. In PR #100 this will include error message in redirect.

I suggest older format with just changing `export.php` to `view.php`.

Reply:
Good catch, fixed it according to the suggestion and made it the same format as in main.